packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -287,23 +315,176 @@
     <SF.Item>
       {{! Notice: we want to emulate the case of a fixed width list }}
       {{! template-lint-disable no-inline-styles }}
-      <ul class="hds-dropdown-list" style="width: 250px">
-        {{#each @model.ITEM_STATES as |state|}}
+      <div class="hds-dropdown__content" style="width: 250px">
+        <ul class="hds-dropdown__list">
+          {{#each @model.ITEM_STATES as |state|}}
+            <Hds::Dropdown::ListItem::CopyItem
+              @text="{{state}}: 91ee1e8ef65b337f0e70d793f456c71d"
+              mock-state-value={{state}}
+              mock-state-selector="button"
+            />
+          {{/each}}
           <Hds::Dropdown::ListItem::CopyItem
-            @text="{{state}}: 91ee1e8ef65b337f0e70d793f456c71d"
-            mock-state-value={{state}}
+            @text="success: 91ee1e8ef65b337f0e70d793f456c71d91ee1e8ef65b337f0e70d793f456c71d91ee1e8ef65b337f0e70d793f456c71d"
+            @isSuccess={{true}}
+            mock-state-value="success"
             mock-state-selector="button"
           />
-        {{/each}}
-        <Hds::Dropdown::ListItem::CopyItem
-          @text="success: 91ee1e8ef65b337f0e70d793f456c71d91ee1e8ef65b337f0e70d793f456c71d91ee1e8ef65b337f0e70d793f456c71d"
-          @isSuccess={{true}}
-          mock-state-value="success"
-          mock-state-selector="button"
-        />
-      </ul>
+        </ul>
+      </div>
       {{! template-lint-enable no-inline-styles }}
     </SF.Item>
   </Shw::Flex>
 
+  <Shw::Text::H4>Header and footer</Shw::Text::H4>
```

Comment:
I think we're missing one example where the header has a separator

Reply:
Updated the examples to showcase what they would look like with or without separators